### ISSUE TITLE
Documentation site navigation is broken

### ISSUE THREAD

**Issue by jesperordrup** [react-bootstrap/react-bootstrap#3233]
Problem:
Clicking navigation items fails silently. After clicking an item you can get on by doing a browser refresh.

When  loading https://react-bootstrap.github.io i get this error in console.

Chrome:
"production-app.js:193 Uncaught TypeError: l.a.hydrate is not a function"

Edge:
 SCRIPT438: SCRIPT438: Object doesn't support property or method 'hydrate'

**Comment by taion** [comment]
@jquense Looks like Gatsby v2 wants to use `ReactDOM.hydrate`, but we're using React 15 for the docs site. Should we upgrade React there or tweak the Gatsby config?

**Comment by jquense** [comment]
We should use react 16, I didnt realize we weren't already, I guess just on the v4 branch

**Comment by esbenvb** [comment]
Until it's fixed, access the docs this way:

```sh
git clone https://github.com/react-bootstrap/react-bootstrap.git
cd react-bootstrap/
git checkout v0.32.0
npm install
npm  run build-docs
cd www/public/
python -m SimpleHTTPServer 8000
```

Read the docs here: http://localhost:8000/

**Comment by esbenvb** [comment]
@nickhaughton why did you downwote my tip?

**Comment by nickhaughton** [comment]
@esbenvb my bad bro

**Comment by dominik-zeglen** [comment]
For some reason, you have to use both `yarn` and `npm` to install dependencies. I don't think it should be working that way.

**Comment by dominik-zeglen** [comment]
@esbenvb this would work only in Python 2. Python 3 equivalent to last command is `python -m http.server 8000`.

**Comment by esbenvb** [comment]
@dominik-zeglen 
> For some reason, you have to use both yarn and npm to install dependencies in docs building process. I don't think it should be working that way.

I think it's been fixed, since it looks like newer versions only use yarn, but I went back to a previous version that works, in the steps I describe above. However the readme is also not up to date in v0.32.0, I had to read the appropriate script name in `package.json` in order to build the docs.

**Comment by dominik-zeglen** [comment]
@esbenvb Thanks for info man :)

**Comment by taion** [comment]
fixed in v0.32.3
